datasets/HMRC-RTS-Small-area/main.py

```diff
@@ -17,15 +18,18 @@
 from gssutils import *
 import json
 
-scraper = Scraper(json.load(open('info.json'))['landingPage'])
-scraper
+info = json.load(open('info.json'))
+scraper = Scraper(seed='info.json')  
+cubes = Cubes('info.json')
+scraper 
 # -
 
-scraper.select_dataset(latest=True)
-scraper
+tabs = {tab.name: tab for tab in scraper.catalog.dataset[0].distribution[1].as_databaker()}
+# tabs = {tab.name: tab for tab in scraper.distribution(latest=True, mediaType=Excel).as_databaker()}
+# excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == ExcelOpenXML]
+# excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == Excel]
 
-tabs = {tab.name: tab for tab in scraper.distribution(title=lambda t: 'Data Tables' in t).as_databaker()}
-#tabs.keys()
+excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == Excel]
```

Comment:
Remove as catolog & distribution has already been selected and excel_distro_list is not used anywhere else in code.

Reply:
Done.